### PR TITLE
:sparkles: support cloudevents for manifestworkreplicaset

### PR DESCRIPTION
**Pull request by skeeey**
## Related issue(s)

Fixes #

### CODE REVIEW

File: pkg/work/hub/manager.go [hunk 1]
```diff
@@ -2,73 +2,166 @@ package hub
...
 )
 
-// RunWorkHubManager starts the controllers on hub.
-func RunWorkHubManager(ctx context.Context, controllerContext *controllercmd.ControllerContext) error {
-	hubWorkClient, err := workclientset.NewForConfig(controllerContext.KubeConfig)
-	if err != nil {
-		return err
+const (
+	sourceID       = "mwrsctrl"
+	sourceClientID = "mwrsctrl-client"
```
Comment (morvencao) [inline pkg/work/hub/manager.go:29]: nit: Is this safe to have the same hard-code clientID for mwrs controller?
does this mean anytime, only one mwrs controller instance is working?

Comment (skeeey) [reply pkg/work/hub/manager.go:29]: maybe need to expose these with flags

File: manifests/cluster-manager/management/cluster-manager-manifestworkreplicaset-deployment.yaml [hunk 2]
```diff
@@ -46,6 +46,7 @@ spec:
         args:
           - "/work"
           - "manager"
+          - "--work-driver=kube"
```
Comment (morvencao) [inline manifests/cluster-manager/management/cluster-manager-manifestworkreplicaset-deployment.yaml:49]: add `--cloudevents-client-id=$(POD_NAME)`?

Comment (skeeey) [reply manifests/cluster-manager/management/cluster-manager-manifestworkreplicaset-deployment.yaml:49]: currently, we make the `cloudevents-client-id` optional, it only need when we use cloudevents, current case is kube, we can omit it

**Comment by skeeey** [comment]
/hold

the sdk-go has a bug, I will fix it, then update the sdk-go again

**Pull request merged**